### ISSUE TITLE
InceptionResnetV2 quantization: block35_1/Relu is lacking min/max data

### ISSUE THREAD

**Issue by chenlh14** [tensorflow/tensorflow#25088]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 16.04.5 LTS
- TensorFlow installed from (source or binary): tensorflow-gpu==1.12.0

**Command as follows**
    tflite_model = tf.contrib.lite.toco_convert(
        frozen_graphdef, [images], [logits], inference_type=tf.contrib.lite.constants.QUANTIZED_UINT8,
        quantized_input_stats=[(127.5, 127.5)])

**Graph as follows**
![graph_run 1](https://user-images.githubusercontent.com/22855898/51511778-926d8480-1e3d-11e9-9f24-29d09fb5c7d9.png)

**Provide the text output from toco_convert**
```
F tensorflow/contrib/lite/toco/tooling_util.cc:1634] Array InceptionResnetV2/InceptionResnetV2/Repeat/block35_1/Relu, which is an input to the MaxPool operator producing the output array InceptionResnetV2/InceptionResnetV2/Mixed_6a/Branch_2/MaxPool_1a_3x3/MaxPool, is lacking min/max data, which is necessary for quantization. If accuracy matters, either target a non-quantized output format, or run quantized training with your model from a floating point checkpoint to change the input graph to contain min/max information. If you don't care about accuracy, you can pass --default_ranges_min= and --default_ranges_max= for easy experimentation.
```

**Comment by shashishekhar** [comment]
During quantization the activations (and weights) are converted to 8 bit. For this conversion the tool needs information for minimum and maximum values. These minimum and maximum values can be obtained 

you can get this information through :
1. [quantized training](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/quantize/README.md), this maybe hard since you may need to change the training code and retrain your model but this will give you better accuracy.
2. The other option is [post training quantization](https://www.tensorflow.org/lite/performance/post_training_quantization] which is simpler but may not give you as good accuracy as with quantized training.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by shashishekhar** [comment]
Closing due to inactivity, please reopen if you are still facing the issue.
